benchmark/net/http_server_for_chunky_client.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+var path = require('path');
+var http = require('http');
+var fs = require('fs');
+var spawn = require('child_process').spawn;
+var common = require('../common.js')
+var test = require('../../test/common.js')
+var pep = path.dirname(process.argv[1])+'/chunky_http_client.js';
+var PIPE = test.PIPE;
+
+var server;
+try {
+  fs.unlinkSync(PIPE);
+} catch (e) { /* ignore */ }
+
+server = http.createServer(function(req, res) {
+  res.writeHead(200, { 'content-type': 'text/plain',
+                       'content-length': '2' });
+  res.end('ok');
+});
+
+server.on('error', function(err) {
+  console.error('Error:');
+  console.error(err);
+});
+
+try {
+  server.listen(PIPE, 'localhost');
```

Comment:
'localhost' is not needed and you may have a race condition here.  The child process should be started from inside the 'listening' event listener.